addons/l10n_se/demo/demo.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="demo_company_se" model="res.company">
+            <field name="name">Swedish Company</field>
+            <field name="country_id" ref="base.se"/>
+            <field name="currency_id" ref="base.SEK"/>
```

Comment:
No need for a Swedish Demo company, although the idea is interesting, but then we need to do it for all of them (admittedly, Argentina has 3)

Reply:
I moved this to a separate branch that will target master.